src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -383,6 +386,16 @@ struct SvgTextNode
     float fontSize;
 };
 
+struct SvgGaussianBlurNode
+{
+    float stdDevX, stdDevY;
+    bool edgeModeWrap;
+};
+
+struct SvgFilterNode
```

Comment:
?

Reply:
This node should enclose other filter nodes and can also contain attributes independently of other filters, such as 'x', 'y', 'width', 'height', and 'units...'.

example:
```
  <filter id="dropShadow" x="2" y="14">
    <feGaussianBlur in="SourceAlpha" stdDeviation="3" />
    <feOffset dx="2" dy="4" />
    <feMerge>
      <feMergeNode />
      <feMergeNode in="SourceGraphic" />
    </feMerge>
  </filter>
  ```

Reply:
What does 'enclose' mean here?  It seems an empty struct and isn't actually used in the logic. If you're adding this for potential future use, please add it only when it's actually needed.